Mise à jour du module "user" et ajout d'un script de base de donnée

RuCaVi opened the pull request:
Le module "user" à été mis à jour afin de pouvoir gérer les e-mails et un script à été ajouté afin de pouvoir insérer des e-mails dans la table "user" de la base de donnée

RuCaVi committed:
Updated user module

RuCaVi committed:
Update readme.md

DidierViret commented:
Merci de tester le fonctionnement avant de faire un Pull request ... le lien "Administration" ne fonctionne plus suite à la mise à jour.
A corriger et vérifier que tout le reste fonctionne bien.

RuCaVi committed:
Revert "Updated user module"

This reverts commit 11bda4abf8154bf2b9dc87a12be5cda0a99ca2b5.

RuCaVi committed:
Revert "Revert "Updated user module""

This reverts commit 390a3c9bd8dff2c88462079b2b8c6da0f1611d82.

RuCaVi committed:
Add email max length to config and link to administration page

RuCaVi commented:
Le liens Administration devrait normalement fonctionner et j'ai également testé les fonctionnalités à l’intérieur de la page au cas où. Il manquait simplement le lien qui redirigeait vers la page Administration.

DidierViret commented:
Intégrer dans ce pull request la modification liée à l'issue #216 car elle concerne aussi une mise à jour du module user.

RuCaVi committed:
Ajout de la possibilité de rendre le champ "user" unique et inversement

RuCaVi committed:
Ajout des scripts de base de données dans le module "user"

Les scripts qui se trouvent dans le module "user" de "ci_packbase_hmvc" ont été ajouter dans le module "user" de "gestion_questionnaires"

RuCaVi committed:
Change "username" index to "username_unique" in SQL script

DidierViret committed:
Add btn_back in user module language file
